test/models/work_test.rb

```diff
@@ -0,0 +1,207 @@
+require "test_helper"
+
+describe Work do
+  # it "does a thing" do
+  #   value(1+1).must_equal 2
+  # end
+
+  describe 'validations' do
+    before do
+      @work = Work.new(
+          category: "book",
+          title: "test work",
+          creator: "test creator",
+          publication_year: 2003,
+          description: "blah blah"
+      )
+      @work2 = Work.new(
+          category: "movie",
+          title: "test work2",
+          creator: "test creator2",
+          publication_year: 2010,
+          description: "blah blah blah"
+      )
+    end
+
+    it 'is valid when all fields are present' do
+      result = @work.valid?
+
+      expect(result).must_equal true
+    end
+
+    it 'is invalid without a title' do
+      @work.title = nil
+
+      result = @work.valid?
+
+      expect(result).must_equal false
+    end
+
+    it 'is invalid if the title and category exists' do
+
+      existing_work = Work.create(
+          category: "book",
+          title: "test work",
+          creator: "lalala",
+          publication_year: 2010,
+          description: "blah blah"
+      )
+
+      result = @work.valid?
+
+      expect(result).must_equal false
+      expect(@work.errors.messages).must_include :title
+
+    end
+
+    it 'is valid even if the title exist but with a different category' do
+
+      existing_work = Work.create(
+          category: "movie",
+          title: "test work",
+          creator: "lalala",
+          publication_year: 2010,
+          description: "blah blah"
+      )
+
+      result = @work.valid?
+
+      expect(result).must_equal true
+
+    end
+  end
+
+  describe 'top_10' do
+    it 'should extract 10 works of the category' do
+
+      category = "book"
+
+      books = Work.top_10(category)
+
+      expect(books.count).must_equal 10
+    end
+
+    it "should extract works even if there are less than 10" do
+      category = "movie"
+
+      movies = Work.top_10(category)
+
+      expect(movies.count).must_equal 1
+    end
+
+    it 'list 0 works if the category does not have any works' do
+      category = "album"
+
+      albums = Work.top_10(category)
+
+      expect(albums.count).must_equal 0
+    end
+
+    it 'list works even if there are no votes' do
+      Vote.delete_all
+
+      category = "book"
+
+      books = Work.top_10(category)
+
+      expect(books.count).must_equal 10
+    end
+
+    it 'list top 10 by number of votes' do
+
+      category = "book"
+
+      books = Work.top_10(category)
+
+      expect(books.first).must_equal works(:book1)
+      expect(books.first.votes.count).must_equal 5
```

Comment:
I also suggest testing that each item has greater than or equal to the number of votes of subsequent works.